api/assessment/assessment.proto

```diff
@@ -70,8 +70,17 @@ service Assessment {
 };
 
 message ListAssessmentResultsRequest {
+  // Optionally, a filter for the cloud service id.  
   string filtered_cloud_service_id = 1;
 
+  enum Compliant {
```

Comment:
can't we just use an `optional bool` here?

Reply:
I thought so too. But, you want either all assessment results or you want to filter for compliant=true or compliant=false. If you don't set the compliant field in the request it is automatically set to false and you cannot distinguish between compliant = false and the compliant field not set.

Reply:
But shouldn’t you get a `*bool` with optional?

Reply:
No, maybe I did something wrong :woman_shrugging:

Reply:
Seems to work:
```
optional bool filtered_compliant = 3;
```

generates 

```
FilteredCompliant *bool  `protobuf:"varint,3,opt,name=filtered_compliant,json=filteredCompliant,proto3,oneof" json:"filtered_compliant,omitempty"`
```

You may need to remove the additional binding (see below), otherwise I think protoc will complain about optional parameters in the query path.

Reply:
Ok, if you don't do it with additional bindings. But with the binding I've only got an bool. 

And why do we need the additional binding for the filtered_cloud_service_id? We also could use the query parameters or not?